docs/tools/api.rst

```diff
@@ -1,48 +1,44 @@
-Aiven API overview
-==================
+Aiven API
+=========
 
-The Aiven API allows you to perform any tasks on Aiven from a client of your choice.
+Use the Aiven API to programmatically perform any task that you can do through the web interface. This is an ideal way to automate tasks involving Aiven at every stage of your workflow.
```

Comment:
```suggestion
Use the Aiven API to perform any programming task that you can perform through the web interface. This is an ideal way to automate tasks, involving Aiven at every stage of your workflow.
```

Reply:
That just repeats the original text, but it clears up what "programmatically" means in this context :-D it's fine the way it was, then.